docs/migrations/web/MIGRATION-v3.md

```diff
@@ -210,6 +211,25 @@ Button `inverted` modifier was removed. Use themes to switch the button colors.
 
 Field Group component children are not aligned to `start` except when the `FieldGroup--fluid` modifier is used.
 
+### Flex: Change in spacing property
+
+The original property `--flex-spacing` has been replaced to allow separate control over spacing on the x and y axes.
+You can now use the properties `--flex-spacing-x` and `--flex-spacing-y` to apply spacing only on the x-axis or y-axis, respectively.
+
+Similarly, when applying responsive spacing for a specific breakpoint, the previous property `--flex-spacing-{breakpoint}` has been replaced.
+You should now use `--flex-spacing-x-{breakpoint}` for horizontal spacing or `--flex-spacing-y-{breakpoint}` for vertical spacing at that specific breakpoint.
+
+#### Migration Guide
+
+To separate spacing on the x and y axes, please manually replace the original `--flex-spacing` property with `--flex-spacing-x` or `--flex-spacing-y`,
+depending on the axis you want to apply the spacing to.
+
+- `<div style="--flex-spacing: var(--spirit-space-1200)">` → `<div style="--flex-spacing-x: var(--spirit-space-1200)">`
```

Comment:
I would add `class="Flex …"` so they know on which element it should be.

```suggestion
- `<div class="Flex …" style="--flex-spacing: var(--spirit-space-1200)">` → `<div class="Flex …" style="--flex-spacing-x: var(--spirit-space-1200)">`
```
What do you think?